<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>paraSet</class>
 <widget class="QMainWindow" name="paraSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>477</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>814</width>
    <height>477</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>814</width>
    <height>477</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="3">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>马达电流（mA）：</string>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Min：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Max：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lEditMotorCurMax">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>50</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lEditMotorCurMin">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QGroupBox" name="groupBox_8">
      <property name="title">
       <string>蓝牙信号（dBm）：</string>
      </property>
      <widget class="QLineEdit" name="lEditRssi">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>40</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Min：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>timeout：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lEditRssiTimeOut">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>60</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>60</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>S</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>工作电流（0.01mA）：</string>
      </property>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Min：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Max：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lEditWorkCurMax">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>40</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lEditWorkCurMin">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lEditWorkCurDly">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>60</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>delay：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>60</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>S</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>充电电流（mA）：</string>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Min：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Max：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lEditChagCurMin">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lEditChagCurMax">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>50</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QGroupBox" name="groupBox_10">
      <property name="title">
       <string>温度范围（℃）：</string>
      </property>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Min：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lEditTemMax">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>50</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lEditTemMin">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_22">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Max：</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="bTnSendPara">
      <property name="text">
       <string>将参数发送给设备</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QGroupBox" name="groupBox_6">
      <property name="title">
       <string>三轴加速度：</string>
      </property>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Min：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lEditAccMax">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>50</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lEditAccMin">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Max：</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QGroupBox" name="groupBox_9">
      <property name="title">
       <string>固件版本：</string>
      </property>
      <widget class="QLineEdit" name="lEditSW">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>3.3V电压值（mV）：</string>
      </property>
      <widget class="QLineEdit" name="lEditVolMin">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lEditVolMax">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>50</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Min：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Max：</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="bTnGetPara">
      <property name="text">
       <string>从设备获取测试参数</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="groupBox_7">
      <property name="title">
       <string>待机电流（0.01mA）：</string>
      </property>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>60</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>S</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lEditSlepCurMax">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>40</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Min：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Max：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lEditSlepCurMin">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>delay：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lEditSlepCurDly">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>60</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QGroupBox" name="groupBox_5">
      <property name="title">
       <string>霍尔状态：</string>
      </property>
      <widget class="QLineEdit" name="lEditHall">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>测试参数设定</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>814</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
